spec/features/delete_draft_work_spec.rb

```diff
@@ -16,18 +16,116 @@
 
   before do
     create(:collection_version_with_collection, collection: collection)
-    sign_in user
   end
 
-  context 'when draft' do
+  context 'when depositor logged in' do
```

Comment:
These first set of tests are the same, I just wrapped in new context block for the depositor being logged in.

Reply:
This is not a new feature, so we don't need a new (slow) feature tests.  Have you considered adding to the policy unit tests instead?